src/main/java/core/basesyntax/files/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.files;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            return Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Don't leave `e.printStackTrace()` or `System.out.println(e.getMessage())` in the catch blocks. Let's rethrow a RuntimeException with an informative message and exception object. For example: `throw new RuntimeException("Can't read file by path: " + filePath, e);`

Reply:
Here is no printStackTrace or System.out.println!